src/components/Datatable/ControlModule/ControlModule.types.ts

```diff
@@ -0,0 +1,5 @@
+import { ControlsConfig } from '../Datatable.types';
+
+export interface ControlModuleProps<D extends ControlsConfig<D>> {
+  someprop?: boolean;
```

Comment:
What is this?

Reply:
removed, it was just a placeholder to test fixing type errors